climakitae/util/generate_gwl_tables.py

```diff
@@ -215,14 +219,19 @@ def get_wl_timestamp(scenario, degree):
                 wl_ts = scenario[scenario >= degree].index[0]
                 return wl_ts
             else:
-                return np.NaN
+                return np.nan
```

Comment:
I know it's an internal function, but for readability can you add type hints here as well?